src/rtc/geogrid.py

```diff
@@ -268,13 +270,56 @@ def get_point_epsg(lat, lon):
         raise ValueError(err_str)
 
 
+def _check_pixel_spacing(y_spacing_positive, x_spacing, epsg, product_str):
+    '''
+    Verify pixel spacing
 
+    Parameters
+    ----------
+    y_spacing_positive: float
+        Y/latitude pixel spacing (absolute value)
+    x_spacing: float
+        X/longitude pixel spacing
+    epsg: int
+        EPSG code
+    product_str: string
+        Product type string (e.g., "Mosaic" or "Bursts")
 
+    Returns
+    -------
+    y_spacing: float
+        Y/latitude pixel spacing
+    x_spacing: float
+        X/longitude pixel spacing
+    '''
+    # Check spacing in X direction
+    if x_spacing is not None and x_spacing <= 0:
+        err_str = f'{product_str} pixel spacing in X/longitude direction must'
+        err_str += f' be positive (x_spacing: {x_spacing})'
+        logger.error(err_str)
+        raise ValueError(err_str)
+    elif x_spacing is None and epsg and epsg == 4326:
+        x_spacing = 0.00027
```

Comment:
Looks like we are repetitively using this constant, which I think is an approximate of `1.0 / 3600.0`. What do you think about defining this somewhere and using it? for example:

```python
SECONDS_IN_DEG = 1.0 / 3600.0
.
.
.
.
x_spacing = SECONDS_IN_DEG
```

Reply:
Thank you for this suggestion, @seongsujeong . I'm setting a constant, but I'm not using `1.0 / 3600.0`, because that number has a repeating decimal and I think it's better to avoid that for pixel spacing. I'll use something like `0.0002777`.